Packs/ReversingLabs_Titanium_Cloud/Integrations/ReversingLabsTitaniumCloudv2/ReversingLabsTitaniumCloudv2.yml

```diff
@@ -25,6 +25,30 @@ configuration:
   - F - Reliability cannot be judged
   type: 15
   required: false
+- display: HTTP proxy address with the protocol and port number.
+  name: http_proxy
+  type: 0
+  required: false
+- display: HTTP proxy username (if needed)
+  name: http_proxy_username
+  type: 0
+  required: false
+- display: HTTP proxy password (if needed)
+  name: http_proxy_password
+  type: 4
```

Comment:
Here too, we recommend using type 9.

Note that you already have a use of type 9 above, in order to distinguish between the two, call the second one a different name.
For example: name: credentinals2, and then in the python you can enter its values like this: 

demisto.params().get("credentials2", {}).get("identifier")